src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -94,28 +102,112 @@ following to integrate with your `my_flutter` app:
 
 3. Run `pod install`.
 
+{{site.alert.note}}
 Whenever you change the Flutter plugin dependencies in `some/path/my_flutter/pubspec.yaml`,
 you need to run `flutter pub get` from `some/path/my_flutter` to refresh the list
 of plugins read by the `podhelper.rb` script. Then run `pod install` again from
 `some/path/MyApp`.
+{{site.alert.end}}
 
-The `podhelper.rb` script ensures that your plugins, the Flutter.framework, and
-the App.framework are embedded in your project.
+The `podhelper.rb` script ensures that your plugins, `Flutter.framework`, and
+`App.framework` are embedded in your project.
 
 You should now be able to build the project using `⌘B`.
 
 ### Under the hood
 
-If you have some reason to do this manually or debug why these steps aren't
-working, here's what's going on under the hood:
-
-- `Flutter.framework` (the Engine library) is getting embedded into your app for you.  This has to match up with the release type
-(debug/profile/release) as well as the architecture for your app (arm*, x86_64, etc.).  CocoaPods pulls this in as a vendored
-framework and makes sure it gets embedded into your native app.
-- `App.framework` (your Flutter application binary) is embedded into your app.  CocoaPods also pulls this in as a vendored framework and
-makes sure it gets embedded into your native app.
-- Any plugins are added as CocoaPod pods.  In theory, it should be possible to manually merge those in as well, but those instructions
-vary on the pod dependencies of each plugin.
-- A build script is added to the Podfile targets that call `install_all_flutter_pods` to ensure that the binaries you build stay up to date
-with the Dart code that's actually in the folder.  It also uses your Xcode build configuration (Debug, Profile, Release) to embed the matching
-release type of `Flutter.framework` and `App.framework`.
\ No newline at end of file
+- `Flutter.framework` (the Engine library) is being embedded into your application by
+CocoaPods as a vendored framework. This has to match up with the release type
+(debug/profile/release) as well as the architecture for your app (arm*, 
+x86_64, etc.).
+- `App.framework` (your Flutter application binary) is being embedded into your
+application by CocoaPods as a vendored framework.
+- Flutter plugins with iOS platform code are added as CocoaPod pods.
+- A build script is added to the Podfile targets that call `install_all_flutter_pods`
+to ensure that the binaries you build stay up to date with the Dart code in
+the Flutter module.  It also uses your Xcode build configuration (Debug, Profile, Release)
+to embed the matching build mode type of `Flutter.framework` and `App.framework`.
```

Comment:
Accurate?
The framework adds a build script to each Podfile target that calls `install_all_flutter_pods`.
This ensures that the binary stays up to date with the Dart code in the Flutter module.